nexteuropa_inpage_nav/js/inpage_nav.js

```diff
@@ -148,7 +148,8 @@
   }
 
   function GetInpageOffsetTop() {
-    return $('.inpage-nav__wrapper').closest('.col-md-3').offset().top;
```

Comment:
@joaocsilva, there's nothing wrong about the fact that you used 'col-md-3' to retrieve the wrapper per se, but this enforces a dependency on bootstrap which is something we might want to avoid.
If you have alternative ways to retrieve that selector without using bootstrap classes please change this.